pamqp/__init__.py

```diff
@@ -2,10 +2,6 @@
 """AMQP Specifications and Classes"""
 __author__ = 'Gavin M. Roy'
 __email__ = 'gavinmroy@gmail.com'
-__since__ = '2011-09-23'
-__version__ = version = '3.1.0'
-
-__all__ = [
```

Comment:
why are you removing `__all__`?

Reply:
Oops, I thought __all__ doesn't work with modules, only with module variables. Put it back.